src/opnsense/mvc/app/models/OPNsense/Firewall/FieldTypes/FilterRuleField.php

```diff
@@ -108,8 +108,13 @@ public function getPriority()
     {
         $configObj = Config::getInstance()->object();
         $interface = (string)$this->interface;
-        if (strpos($interface, ",") !== false || empty($interface)) {
-            // floating (multiple interfaces involved)
+
+        if (
+            // floating: empty, multiple, or inverted interface
+            (string)$this->interfacenot === "1" ||
+            (strpos($interface, ",")) !== false ||
+            empty($interface)
```

Comment:
this doesn't seem to match our actual sorting

https://github.com/opnsense/core/blob/c6eaefc1e26550085b09abf1cca73fdc8f76db13/src/etc/inc/filter.lib.inc#L675-L681

Reply:
https://github.com/opnsense/core/commit/be0b18930f36f9 says it's a floating rule pinned to one interface so this seems correct?

Reply:
`(string)$this->interfacenot === "1" ||` but this adds single interface ones ;)